volttrontesting/platform/test_basehistorian.py

```diff
@@ -13,7 +13,12 @@
 from volttron.platform.vip.agent.subsystems.query import Query
 from volttrontesting.utils.utils import AgentMock
 from time import sleep
-from volttrontesting.platform.test_instance_setup import create_vcfg_vhome
+
+from volttron.platform import is_rabbitmq_available
+if is_rabbitmq_available():
+    from volttrontesting.platform.test_instance_setup import create_vcfg_vhome
+else:
+    pytest.skip("Pika is not installed", allow_module_level=True)
```

Comment:
the create_vcfg_vhome method seem to be create a volttron home and nothing else. (The method should be name create_vhome )So I don't see why it should depend on pika install